test/unit/user_test.rb

```diff
@@ -109,20 +109,14 @@ class UserTest < ActiveSupport::TestCase
     assert_equal "heythere", user.password
   end
 
-  test "appreciations_received appreciations_sent feedback_received feedback_sent" do
+  test "appreciations_received appreciations_sent" do
```

Comment:
[pronto-rubocop warning] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

Reply:
Going to ignore this since this style is all over the place.